### ISSUE TITLE
none of the samples work

### ISSUE THREAD

**Issue by ethanneff** [Microsoft/reactxp#637]
[hello-world](https://github.com/Microsoft/reactxp/tree/master/samples/hello-world)
```sh
$ yarn web-watch
ERROR in [at-loader] ./src/MainPanel.tsx:83:22
    TS2339: Property 'useSafeInsets' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<View> & Readonly<{ children?: ReactNode; }> & Read...'.

ERROR in [at-loader] ./src/SecondPanel.tsx:82:22
    TS2339: Property 'useSafeInsets' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<View> & Readonly<{ children?: ReactNode; }> & Read...'.
```

[TodoList](https://github.com/Microsoft/reactxp/tree/master/samples/TodoList)
```sh
$ yarn web-watch
ERROR in [at-loader] ./src/LocalDb.ts:84:9
    TS2322: Type 'Promise<Promise<Todo[]>>' is not assignable to type 'Promise<Todo[]>'.
  Types of property 'catch' are incompatible.
    Type '(errorFunc: ErrorFunc<Promise<Todo[]>>) => Promise<Promise<Todo[]>>' is not assignable to type '(errorFunc: ErrorFunc<Todo[]>) => Promise<Todo[]>'.
      Types of parameters 'errorFunc' and 'errorFunc' are incompatible.
        Type 'Todo[] | Thenable<Todo[]>' is not assignable to type 'Promise<Todo[]> | Thenable<Promise<Todo[]>>'.
          Type 'Todo[]' is not assignable to type 'Promise<Todo[]> | Thenable<Promise<Todo[]>>'.
            Type 'Todo[]' is not assignable to type 'Thenable<Promise<Todo[]>>'.
              Property 'then' is missing in type 'Todo[]'.

ERROR in [at-loader] ./src/EditTodoPanel.tsx:42:22
    TS2339: Property 'useSafeInsets' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<View> & Readonly<{ children?: ReactNode; }> & Read...'.

ERROR in [at-loader] ./src/EditTodoPanel.tsx:67:21
    TS2339: Property 'accessibilityId' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<TextInput> & Readonly<{ children?: ReactNode; }> &...'.

ERROR in [at-loader] ./src/TodoListPanel.tsx:31:22
    TS2339: Property 'useSafeInsets' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<View> & Readonly<{ children?: ReactNode; }> & Read...'.
```

**Comment by ethanneff** [comment]
thanks!

**Comment by erictraut** [comment]
I just published reactxp 1.2.0. (I decided to bump the middle number rather than call it 1.1.2.) All of the reactxp samples are now updated to use the latest published version.
